libraries/botbuilder-vendors/src/index.ts

```
@@ -0,0 +1,75 @@
+            logger.package.header({ name: pkg.name });
+
+            const pkgDir = path.dirname(absPath);
+            const directory = pkg.localDependencies!.__directory ?? 'vendors';
+            delete pkg.localDependencies!.__directory;
+
+            const { vendors, dependencies, unknown } = await findByPackage(pkg, allVendors);
+            const newVendors = [...unknown, ...vendors];
+
+            logger.package.vendors.header({ vendors: newVendors.length });
+
+            await install({ vendors: newVendors, dependencies, pkgDir, directory, shouldSetDependencies: flags.setDependencies });
```

ESLint

[error] Replace `·vendors:·newVendors,·dependencies,·pkgDir,·directory,·shouldSetDependencies:·flags.setDependencies` with `⏎················vendors:·newVendors,⏎················dependencies,⏎················pkgDir,⏎················directory,⏎················shouldSetDependencies:·flags.setDependencies,⏎···········`

(prettier/prettier)